tensorflow/core/ops/mkl_nn_ops.cc

```diff
@@ -1140,6 +1140,72 @@ MKL-DNN implementation of quantized depthwise Conv2D with Bias, Relu and Requant
 expected to invoke this operator.
 )doc");
 
+REGISTER_OP("_MklFusedBatchNormV3")
+    .Input("x: T")
+    .Input("scale: U")
+    .Input("offset: U")
+    .Input("mean: U")
+    .Input("variance: U")
+    .Input("mkl_x: uint8")
+    .Input("mkl_scale: uint8")
+    .Input("mkl_offset: uint8")
+    .Input("mkl_mean: uint8")
+    .Input("mkl_variance: uint8")
+    .Output("y: T")
+    .Output("batch_mean: U")
+    .Output("batch_variance: U")
+    .Output("reserve_space_1: U")
+    .Output("reserve_space_2: U")
+    .Output("reserve_space_3: U")
+    .Output("mkl_y: uint8")
+    .Output("mkl_batch_mean: uint8")
+    .Output("mkl_batch_variance: uint8")
+    .Output("mkl_reserve_space_1: uint8")
+    .Output("mkl_reserve_space_2: uint8")
+    .Output("mkl_reserve_space_3: uint8")
+    .Attr("T: {bfloat16, float}")
```

Comment:
The normal TF version also supports `T = half`. Do you need to check for `half` during graph rewrite? Same for the grad op.

Reply:
Done